### DISCUSSION TITLE
Sözel mantık sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#195] [Zeka Sorusu] [answered]
Balık tutmaktan dönen Ali “Tuttuğum her balık alabalıktı. Gördüğüm her alabalığı tuttum.” demiştir.
Buna göre aşağıdaki ifadelerden hangisi doğrudur.
A ) Ali balık tutmaya gittiğinde sadece alabalık görmüştür
B ) Ali’nin balık tuttuğu bölgede alabalıktan başka tür balık bulunmamaktadır
C ) Ali gördüğü her balığı tutmuştur
D ) Ali balık tutmaya gittiğinde alabalıktan başka balık görmemiştir
E ) Ali alabalıktan başka balık tutmamıştır

**Comment by KayraMertAydin** [comment, marked as answer]
E
"Tuttuğum her balık alabalıktı" diyor. "Tuttuğum her balık" derken kovasındaki tüm balıkları kastediyor, "... alabalıktı." yani kovasında alabalıktan başka balık yoktur.
